src/applications/appeals/10182/constants.js

```diff
@@ -9,7 +9,8 @@ export const PROFILE_URL = '/profile';
 export const DECISION_REVIEWS_URL = '/decision-reviews/';
 export const FACILITY_LOCATOR_URL = '/find-locations';
 export const CONTESTED_CLAIMS_URL = '/decision-reviews/contested-claims';
-export const BOARD_APPEAL_OPTIONS_URL = '/decision-reviews/board-appeal/';
+export const BOARD_APPEAL_OPTIONS_URL =
+  '/decision-reviews/board-appeal/#what-are-my-board-appeal-optio';
```

Comment:
should `optio` be `option` or `options`?

Reply:
The anchor id's are truncated 😿 
![Screen Shot 2021-06-01 at 2 12 10 PM](https://user-images.githubusercontent.com/136959/120377956-6acb3880-c2e3-11eb-9cde-9fc2cb383a7b.png)